README.md

```diff
@@ -35,13 +35,29 @@ options.
 or you can use it from the repository root to operate on all the subprojects
 together.
 
+### Targeting Options
+
+Many tasks which operate on multiple projects accept targeting options, which
+generally select or filter the command to a subset of the subprojects.
+
+- `:in <names>`             Add the named projects directly to the targets.
+- `:upstream`               Add the transitive dependencies of the current project to the targets.
+- `:upstream-of <names>`    Add the transitive dependencies of the named projects to the targets.
+- `:downstream`             Add the transitive consumers of the current project to the targets.
+- `:downstream-of <names>`  Add the transitive consumers of the named projects to the targets.
+- `:select <key>`           Use a selector from the config to filter target projects.
+- `:skip <names>`           Exclude one or more projects from the target set.
+
+Each `<names>` argument can contain multiple comma-separated project names, and
+all the targeting options may be provided multiple times.
```

Comment:
Might be worth mentioning that projects named `a/b` can be referred to by `a/b` or `b`.